fragmenttestrule/src/main/java/com/android21buttons/fragmenttestrule/FragmentTestRule.java

```diff
@@ -242,4 +246,45 @@ public F getFragment() {
         }
         return fragment;
     }
+
+    /**
+     * Helper method that allows to call methods on your fragment under test.
+     * <p>
+     * Internally it calls {@link Instrumentation#runOnMainSync(Runnable)}, executing the method {@link Runner#run(Object)} with the
+     * fragment under test as the parameter.
```

Comment:
I don't like to explain the internals. I think that we just need to say that this method blocks the current thread until the code inside the Runner is executed. This is important if someone wants to know the difference between this method and `runOnUiThread`.